includes/admin/admin-actions.php

```diff
@@ -83,4 +83,17 @@ function edd_display_product_tabs() {
 	</h2>
 	<br />
 	<?php
-}
\ No newline at end of file
+}
+
+/**
+ * Loads SVG icons into the admin footer.
+ * 
+ * @since 2.9.x
+ */
+function edd_svg_admin_icons() {
```

Comment:
This function could one day be responsible for outputting any SVG icon into the admin (should we need to), hence why it's in this file. Let me know if there's a better place. 

Also, am I correct in saying we don't have a helper function that holds the default payment methods? We have `edd_get_option( 'accepted_cards', array() );` but that's only the methods saved. Ideally I'd like to pass in only the default payment method keys to `edd_svg_icons()` and not anything else.